simulation_parameters/microbe_stage/organelles.json

```diff
@@ -608,7 +608,7 @@
     "Processes": {},
     "Components": {
       "Storage": {
-        "Capacity": 8
+        "Capacity": 3
```

Comment:
I'm going to bump this back up so that vacuole would actually feel really impactful, because I'm doubting that when storage is made into a problem but the solution is also nerfed really heavily, it still won't feel worth it to add a vacuole.